source/rust_verify/src/driver.rs

```diff
@@ -253,10 +253,10 @@ where
 {
     if !build_test_mode {
         if let Some(VerusRoot { path: verusroot, in_vargo }) = find_verusroot() {
-            verifier
-                .args
-                .import
-                .push((format!("vstd"), verusroot.join("vstd.vir").to_str().unwrap().to_string()));
+            if !verifier.args.no_vstd {
+                let vstd = verusroot.join("vstd.vir").to_str().unwrap().to_string();
+                verifier.args.import.push((format!("vstd"), vstd));
+            }
```

Comment:
@utaal This is independent of the rest of the commit but it came up during testing -- I hope this is the correct change to disable loading of vstd with --no_vstd.